tests/integration/test_service_manager.py

```diff
@@ -84,3 +112,13 @@ def _validate_if_service_running_by_making_rpc(port_number):
     with grpc.insecure_channel("localhost:" + port_number) as channel:
         stub = measurement_service_pb2_grpc.MeasurementServiceStub(channel)
         stub.GetMetadata(measurement_service_pb2.GetMetadataRequest())  # RPC call
+
+
+def _get_marker_value(request, marker_name, default=None):
```

Comment:
type hints

Reply:
Removed this method since it is no longer needed.